lib/lightning_web/live/profile_live/form_component.ex

```diff
@@ -4,32 +4,23 @@ defmodule LightningWeb.ProfileLive.FormComponent do
   """
   use LightningWeb, :live_component
 
-  alias Lightning.Accounts.User
   alias Lightning.Accounts
 
   @impl true
-  def update(%{user: user} = assigns, socket) do
+  def update(%{user: user, action: action}, socket) do
     {:ok,
      socket
-     |> assign(:password_changeset, Accounts.change_user_password(user))
      |> assign(
-       :email_changeset,
-       Map.put(Ecto.Changeset.change(%User{}), :valid?, false)
-     )
-     |> assign(:user, user)
-     |> assign(assigns)}
+       password_changeset: Accounts.change_user_password(user),
+       email_changeset: user |> Accounts.validate_change_user_email(%{}),
```

Comment:
@mtuchi You didn't need a blank changeset here, you generally use the same changeset that is used in the validate step except with no params to start with.